source/common/quic/quic_transport_socket_factory.cc

```diff
@@ -1,9 +1,14 @@
 #include "source/common/quic/quic_transport_socket_factory.h"
 
+#include <memory>
+
 #include "envoy/extensions/transport_sockets/quic/v3/quic_transport.pb.validate.h"
 
+#include "source/common/quic/envoy_quic_proof_verifier.h"
```

Comment:
Are these includes sorted correctly? (Maybe they are, but I would have expected to see both "source/extensions/transport_sockets/" includes together)